cypress/support/pages/contactForm.pageObject.js

```diff
@@ -19,11 +19,11 @@ class ContactFormPageObject extends PageObject {
     return cy.contains('.btn', 'Send message');
   }
 
-  typeEmail(email) {
+  typeEmail() {
```

Comment:
The `typeEmail` method is missing the `email` parameter. Add `email` as a parameter to this method and use it in `this.emailField.type(email, { force: true });`.